lte/gateway/c/core/oai/tasks/sgw_s8/sgw_s8_handlers.cpp

```diff
@@ -1476,7 +1486,7 @@ void sgw_s8_handle_s11_create_bearer_response(
     itti_s11_nw_init_actv_bearer_rsp_t* s11_actv_bearer_rsp, imsi64_t imsi64) {
   OAILOG_FUNC_IN(LOG_SGW_S8);
   uint32_t msg_bearer_index = 0;
-  sgw_eps_bearer_ctxt_t dedicated_bearer_ctxt = {0};
```

Comment:
Why are these changes necessary? It seems that it would be safer to keep the initialization?

Reply:
> Why are these changes necessary? It seems that it would be safer to keep the initialization?

Hi @LKreutzer, Thanks for reviewing the PR
The structure, sgw_eps_bearer_ctxt_t contains nested structure, if these structures contains enumeration structures, the compiler reports an error saying that "invalid conversion from ‘int’ to enum type"
62 typedef struct sgw_eps_bearer_ctxt_s {
 63   paa_t paa;
 64   ebi_t eps_bearer_id;  ///< An EPS bearer identity uniquely identifies an EPS
 65                         ///< bearer for one UE accessing via E-UTRAN
 66   traffic_flow_template_t tft;  ///< Traffic Flow Template
 67
 68   ip_address_t
 69       p_gw_address_in_use_up;
 <contains few more fields>
 }

To fix the compiler, we initialized with empty braces and verified with unit test cases that it gets initialized with zero values
![image](https://user-images.githubusercontent.com/30685618/182341649-ca3859c9-6c05-424b-8084-3b828fb38a35.png)